remoting/tests/src/test/java/org/jboss/as/remoting/RemotingCLITestCase.java

```diff
@@ -0,0 +1,22 @@
+package org.jboss.as.remoting;
+
+import org.jboss.as.cli.scriptsupport.CLI;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+@Ignore // This should be run with started EAP
```

Comment:
What is the point of adding this test if it is ignored?

I suppose this test should be moved to the basic integration test suite.

Reply:
@jmesnil You mean here: https://github.com/wildfly/wildfly-core/tree/master/testsuite/standalone or https://github.com/wildfly/wildfly/tree/master/testsuite/integration ?

Reply:
Fixed